packages/manager/modules/pci/src/projects/new/payment/payment.html

```diff
@@ -0,0 +1,98 @@
+<div class="pci-projects-new-payment">
+
+    <div class="text-center"
+         data-ng-if="$ctrl.loading.init">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <div data-ng-if="!$ctrl.loading.init"
+         data-ng-switch="$ctrl.paymentType">
+
+        <div data-ng-switch-when="paymentMethod">
+            <pci-project-new-payment-default
+                data-ng-if="$ctrl.defaultPaymentMethod"
+                data-default-payment-method="$ctrl.defaultPaymentMethod">
+            </pci-project-new-payment-default>
+
+            <pci-project-new-payment-add
+                data-ng-if="!$ctrl.defaultPaymentMethod"
+                data-payment-types="$ctrl.paymentTypes">
+            </pci-project-new-payment-add>
+        </div>
+
+        <div data-ng-switch-when="credits">
+            <pci-project-new-payment-credits></pci-project-new-payment-credits>
+        </div>
+
+        <div class="row">
+            <div class="col-md"
```

Comment:
Missing column size

Reply:
As what I understand from bs4 grid documentation, this kind of class is valid: https://getbootstrap.com/docs/4.3/layout/grid/#how-it-works

> Thanks to flexbox, grid columns without a specified width will automatically layout as equal width columns. For example, four instances of .col-sm will each automatically be 25% wide from the small breakpoint and up. See the auto-layout columns section for more examples.